x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/steps/get_started.tsx

```diff
@@ -51,16 +54,37 @@ const GettingStartedStepContent: React.FunctionComponent<QuickStartCreateForm> =
   submit,
 }) => {
   const { getHref } = useLink();
+  const theme = useTheme() as EuiTheme;
 
   const fleetServerHostsOptions = useMemo(
-    () =>
-      fleetServerHosts.map((fleetServerHost) => {
+    () => [
+      ...fleetServerHosts.map((fleetServerHost) => {
         return {
-          text: fleetServerHost.name,
+          inputDisplay: `${fleetServerHost.name} (${fleetServerHost.host_urls[0]})`,
           value: fleetServerHost.id,
         };
       }),
-    [fleetServerHosts]
+      {
+        icon: <EuiIcon type="plus" size="m" color="primary" />,
+        inputDisplay: (
+          <FormattedMessage
+            id="xpack.fleet.fleetServerSetup.addFleetServerHostBtn"
+            defaultMessage="Add new Fleet Server Hosts"
+          />
+        ),
+        dropdownDisplay: (
+          <EuiText size="relative" color={theme.eui.euiColorPrimary}>
+            <FormattedMessage
+              id="xpack.fleet.fleetServerSetup.addFleetServerHostBtn"
+              defaultMessage="Add new Fleet Server Hosts"
+            />
+          </EuiText>
+        ),
+        'data-test-subj': 'fleetServerSetup.addNewHostBtn',
+        value: '@@##ADD_FLEET_SERVER_HOST##@@',
```

Comment:
Seems we have some duplication here. Think it's possible to abstract this UI or too challenging with the differing data requirements?